### ISSUE TITLE
Add metabox on the "Quick Edit" screen

### ISSUE THREAD

**Issue by kiarashi** [WebDevStudios/CMB2#524]
I've built an extra metabox for a custom post_type with a checkbox where an author can select if the post will be marked as a featured item or not. 

I was wondering if it was possible within CMB2 to show this metabox of mine on the "Quick Edit" screen? And if the metabox can also be added as an extra column. 
Similar to the columns Author, Comments, and Date on the All Pages page under the post_type Page (../edit.php?post_type=page page)

Hope it makes sense what I'm trying to tell. If not, I'll use screenshots to explain. ^^;

**Comment by jajouka79** [comment]
did you ever figure this out? I need to customise the quick edit and bulk edit options adding cmb2 fields, anyone know how to do this?

**Comment by jajouka79** [comment]
http://wpdreamer.com/2012/03/manage-wordpress-posts-using-bulk-edit-and-quick-edit/#add_to_bulk_quick_edit

this article contains the info to do it without cmb2 but for an installation that depends on cmb2 none of this will work

**Comment by kiarashi** [comment]
@jajouka79: I'm sorry for the late reply. 
I'm afraid I wasn't able to make this work either. :(

**Comment by jtsternberg** [comment]
Closing as duplicate of #95
